src/chat.rs

```diff
@@ -1051,8 +1051,11 @@ impl ChatId {
                     self,
                     MessageState::OutPreparing,
                     MessageState::OutDraft,
-                    MessageState::OutPending,
```

Comment:
So, this makes the w/a with `self_added` check necessary only for the older DC versions. Checked by hand with `test_0_complex_or_slow.py::TestGroupStressTests::test_synchronize_member_list_on_group_rejoin`